docs/UserGuide.md

```diff
@@ -176,17 +235,60 @@ If you are using a PDF version of this document, be careful when copying and pas
 
 <br>
 
-## Command Overview
+## **Command Overview**
+
+The following sections describe the various commands you can use in the DocTrack application.
+
+---
+
+<a href="#person-commands" class="badge bg-dark">Person commands</a>
+Click this section to know more about the commands, or click on the individual commands below for information 
+specific to those commands.
+
+| Action                                                      | Format                                                                                    | Examples                                                                                                       |
+|-------------------------------------------------------------|-------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------|
+| **[Add person](#adding-a-person-add-person)**               | `add person n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS st/STATUS [t/TAG]…​`                  | `add person n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 st/recovering t/friend` |
+| **[List all persons](#listing-all-persons-list-person)**    | `list person`                                             <br/>                           | `list person`                                                                                                  |
+| **[Edit person](#editing-a-person-edit-person)**            | `edit person INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [st/STATUS] [t/TAG]…​` | `edit person 2 n/James Lee e/jameslee@example.com`                                                             |
+| **[Find person](#finding-persons-by-name-find-person)**     | `find person KEYWORD [MORE_KEYWORDS]`                                                     | `find person n/James Jake`                                                                                     |
```

Comment:
Under the `find person` feature, can we remove line 384 `- Use the prefix `n/` to search for names.`, since it is already in the table of prefixes?

Reply:
I think it's good to keep, because it makes it easier to skim!